Python/thread.c

```diff
@@ -247,47 +248,114 @@ find_key(int set_value, int key, void *value)
     return p;
 }
 
-/* Return a new key.  This must be called before any other functions in
+
+/* Thread Local Storage (TLS) API, DEPRECATED since Python 3.7
+
+   In the own implementation, TLS API has been changed to call TSS API.
```

Comment:
For the second sentence here, I'd suggest "While the native thread-local storage implementations have been retained, this fallback implementation is now just a compatibility wrapper around the TSS API."